rhoas-cli-service-registry/README.adoc

```diff
@@ -0,0 +1,371 @@
+  "owner": "User",
+  "status": "accepted",
+  "updated_at": "2021-09-27T15:35:01Z"
+}
+----
+
+[NOTE]
+====
+If you do not want to use the default values,
+enter the following command: `rhoas service-registry create`.
+You will be prompted to enter the `Name` for the Service Registry instance.
+====
+--
+
+. Enter the following command to verify that the Service Registry instance is ready to use.
++
+--
+This command shows that the Service Registry instance is ready to use,
+because the `Status` field is `ready`.
+
+.Reviewing details of a Registry instance
+[source,shell]
+----
+$ rhoas status service-registry
+
+  Service Registry
+  -----------------------------------------------------------------------------------------------------------------------------------------------------------------
+  ID:                    63620903-4a29-4ba4-8229-7ab793bf73f3
+  Name:                  my-registry
+  Status:                ready
+  Registry URL:          https://service-registry-service-registry-production.apps.app-sre-prod-04.i5h0.p1.openshiftapps.com/t/63620903-4a29-4ba4-8229-7ab793bf73f3
+----
+
+[NOTE]
+====
+When you created the Service Registry instance, it was set as the current instance automatically.
+If you have multiple Service Registry instances,
+you can switch to a different instance by using the `rhoas service-registry use` command.
+====
+--
+
+[id="proc-creating-service-registry-account_{context}"]
+== Creating a service account
+
+[role="_abstract"]
+To connect your applications or services to a Service Registry instance, you must first create a service account with credentials.
+The credentials are exported to a file on your computer,
+which you can use to authenticate your application with your Service Registry instance.
+
+.Prerequisites
+
+* You have created a Service Registry instance, and it has a `ready` status.
+
+.Procedure
+
+. Enter the following command to create a service account.
++
+--
+This example creates a service account called `my-service-acct` and saves the credentials in a JSON file.
+
+.Creating a service account
+[source,shell]
+----
+$ rhoas service-account create --name "my-service-acct" --file-format json
+
+Service account my-service-account created
+Credentials saved to /home/developer/my-project/credentials.json
+----
+
+Your service account credentials are created and saved to a JSON file.
+
+[NOTE]
+====
+When creating a service account, you can choose the file format and location to save the credentials.
+For more information, see the `rhoas service-account create` command help.
+====
+--
+
+. To verify your service account credentials,
+view the `credentials.json` file that you created.
++
+--
+.Verifying service account credentials
+[source,shell]
+----
+$ cat credentials.json
+{
+	"clientID":"srvc-acct-8c95ca5e1225-94a-41f1-ab97-aacf3df1",
+	"clientSecret":"facf3df1-ab97-2253-aa87-ab97",
+  "oauth_token_url": "https://identity.api.openshift.com/auth/realms/rhoas/protocol/openid-connect/token"
```

Comment:
Just service registry with oauth? Not Kafka?

Reply:
Good catch! Not sure why we even print those outputs, but let's leave them - I tried to replicate what kafka had